voluncheer/opportunityboard/test_forms.py

```diff
@@ -0,0 +1,69 @@
+from datetime import timedelta
+
+from django.test import TestCase
+from django.utils import timezone
+
+from opportunityboard.forms.postanopportunity import PostAnOpportunityForm
+from opportunityboard.models import Opportunity
+from profiles.models import Organization
+from profiles.models import User
+from profiles.models import UserType
+
+
+class PostAnOpportunityFormTest(TestCase):
+    """Test cases for Post An Opportunity form"""
+
+    def setUp(self):
+        """See base class."""
+        self.form = PostAnOpportunityForm()
+        self.user = Organization.objects.create(
+            user=User.objects.create(
+                email="jedi@jedi.com",
+                password="peace_and_justice_for_the_galaxy",
+                type=UserType.ORGANIZATION,
+            ),
+            name="Jedi Council",
+        )
+        self.user.save()
+
+    def test_validation(self):
+        """Test Post An Opportunity validation"""
+        data = {
```

Comment:
This can go into a built-in Django test function called `setUp(self)`, which create an opportunity on the test db for each test to use.

Reply:
Yeah. I'll try add setUp